src/configuration/rendererSettings.js

```diff
@@ -1,26 +1,26 @@
 export default {
     //query variables
-    form_col: 'Datastr',
-    formDescription_col: 'Form',
-    field_col: 'Field Name',
-    fieldDescription_col: 'Field',
-    marking_group_col: 'Query Open By: Marking Group',
-    visit_col: 'Visit/Folder',
+    form_col: 'formoid',
+    formDescription_col: 'ecrfpagename',
+    field_col: 'fieldname',
+    fieldDescription_col: 'fieldname', //there is not a dscriptive column in the test data prescribed by heather
+    marking_group_col: 'markinggroup',
+    visit_col: 'folderoid',
 
     //query age settings
-    age_col: 'Query Age',
+    age_col: 'qdays',
     age_category_col: 'Query Age Category',
```

Comment:
we should ditch this setting since we're deriving the variable rather than expecting it